### ISSUE TITLE
Unavailable PriorityQueue method documentation

### ISSUE THREAD

**Issue by eduherminio** [dotnet/runtime#54779]
### Description

I can't see `PriorityQueue<TElement, TPriority>` method documentation, neither hovering over the methods or going to its 'definition' (f12, see attached picture).

In the same environment, I've tested `Queue<T>` and `Stack<T>` and both of them work as expected.

![image](https://user-images.githubusercontent.com/11148519/123516491-93eda780-d69c-11eb-9c57-78c26f9d45c5.png)

### Configuration

```
6.0.100-preview.5.21302.13

Microsoft Visual Studio Community 2022 Preview (64-bit)
Version 17.0.0 Preview 1.0
```

**Comment by stephentoub** [comment]
PriorityQueue is new in .NET 6. I believe we haven't yet imported the XML comments for anything Intelligence. to seed the docs and IntelliSense. @jeffhandley?

**Comment by eiriktsarpalis** [comment]
That's probably right, the api docs repo currently only contains a [skeleton document for the type](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Collections.Generic/PriorityQueue%602.xml). IIRC the initial plan was for triple-slash docs to be our source of truth, but it seems like for 6 we'd need to port new API docs manually. cc @carlossanlop

**Comment by jeffhandley** [comment]
Correct -- the XML docs will be the source of truth this release, and we haven't yet produced a release with the XML docs ingested into IntelliSense. I believe that will happen with RC1; is that right, @safern & @carlossanlop?

**Comment by eiriktsarpalis** [comment]
I believe it is up to area owners to manually port XML docs for their share of new APIs. @carlossanlop has already created issues for each area.


Closing in favor of https://github.com/dotnet/runtime/issues/54869.

**Comment by safern** [comment]
We always update our intellisense xml files on the RC* releases once the API surface for the current TFM is complete. So that will give time to owners to add docs for new APIs to the dotnet/docs repo and then we can update the intellisense package.
